assignment/src/com/assignment3/LambdaEmploye/LambdaAndStreams.java

```diff
@@ -0,0 +1,36 @@
+package com.assignment3.LambdaEmploye;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class LambdaAndStreams {
+    public static void main(String[] args) {
+
+        //Creating a List of type Employee
+        List<Employee> employeeList = new ArrayList<Employee>();
+
+        //Appending values to the list
+        employeeList.add(new Employee(1,"CSE", 20000));
+        employeeList.add(new Employee(2,"CSE", 30000));
+        employeeList.add(new Employee(3,"IT", 20000));
+        employeeList.add(new Employee(4,"EEE", 40000));
+        employeeList.add(new Employee(5,"IT", 10000));
+        employeeList.add(new Employee(5,"EEE", 50000));
+
+        //Filtering employeeList with department and storing them in separate department Lists
+        List<Integer> employeeCSE = employeeList.stream().filter(obj -> obj.department.equals("CSE")).map(obj1 -> obj1.salary).collect(Collectors.toList());
```

Comment:
Write separate method for find each group's employees, max salary and to print. eg: findMaxForCSE(), etc.